hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTestServer.cpp

```diff
@@ -449,6 +439,20 @@ bool ChangesetSplitDeleteTestServer::respond(HttpConnection::HttpConnectionPtr&
   return continue_processing && !get_interupt();
 }
 
+bool TimeoutTestServer::respond(HttpConnection::HttpConnectionPtr &connection)
+{
+  //  Read the HTTP request
+  parse_request(connection);
+  //  Wait for 2 seconds to cause a timeout on the other end
+  std::this_thread::sleep_for(std::chrono::seconds(2));
```

Comment:
Timeout unit test sets the timeout to 1 second so that 2 seconds is too long to wait and it triggers the timeout error response.